orm/bucket.go

```diff
@@ -203,14 +210,24 @@ func (b Bucket) Sequence(name string) Sequence {
 //
 // Designed to be chained.
 func (b Bucket) WithIndex(name string, indexer Indexer, unique bool) Bucket {
+	return b.WithMultiKeyIndex(name, func(obj Object) ([][]byte, error) {
```

Comment:
Question: why use Indexed interface then only have one concrete implementation?

I guess you want to generalize this when pr finished?

Reply:
Then again, MultiIndexer is a superset of Indexer, so you can get away with this